std/stdio.d

```diff
@@ -107,7 +107,11 @@ version (Windows)
 {
     private alias FSChar = wchar;
 }
-else version (Posix)
+else version (linux)
```

Comment:
Why not just keep as `else version (Posix)`?

Reply:
https://github.com/dlang/phobos/pull/7500#discussion_r431447218

Reply:
So more clearly, for my opinion

Reply:
Yeah, this needs to be switched back. Just add the case for your new situation, and don't change the existing situations.

Reply:
done